releases.md

```diff
@@ -38,6 +38,65 @@ Further documentation available:
 - [Installing Tekton][tekton-installation]
 - Standard for [release notes][release-notes-standards]
 
+## Backwards Incompatible Changes
+
+In general we aim for new features to be added in a backwards compatible way,
+but sometimes we will need to make breaking changes. This policy outlines how we
+will make and communicate these.
+
+**NOTE**: Tekton Chains is working towards a formal beta release. Until then,
+all features are technically considered `alpha` (though we continue to do our
+best to retain backwards compatibility).
+
+### Stability levels
+
+| Stability Level | Deprecation Window                                       |
+| --------------- | -------------------------------------------------------- |
+| alpha           | none                                                     |
+| beta            | 3 months or 3 minor releases (which ever is longer)      |
+| stable          | 12 months or 3 LTS minor releases (which ever is longer) |
+
+Example: if a beta feature is announced as deprecated in v0.10.0, it can be
+completely removed in v0.13.0.
+
+### What's in scope
+
+- [User Configuration](docs/config.md)
+- Input mechanisms from Pipelines (e.g.
+  [Type Hinting](https://github.com/tektoncd/chains/blob/main/docs/config.md#chains-type-hinting))
+- Chains generated provenance output formats
+- Exported Client Libraries (https://pkg.go.dev/github.com/tektoncd/chains)
+
+<!-- TODO(wlynch): if/when config type aliases are a thing, document policy around how these can change -->
+
+#### Exceptions
+
+We reserve the right to make breaking changes (regardless of stability level)
+under certain conditions:
+
+- Security
+
+  We will make breaking changes for security reasons if necessary.
```

Comment:
Suggest adding CVE or any other vulnerabilities to be a little more specific in addition to `security`.

Reply:
Done.